pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -209,6 +209,9 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		)
 	}
 
+	// Needs to be set in BeforeEach in order to not expose storageClass/Profile informer/store
+	sanityExecute := func() {}
+
```

Comment:
Alternative solution: `storageProfileInformer` and `storageClassStore` as variables in the test file

Reply:
I personally don't prefer it as it gives wrong impression that you can use them in tests (the informer)